Packaging/Debian/nextspace-applications-0.85+20200520.gitdcb7a3e6/debian/patches/fix-environment-info.patch

```diff
@@ -0,0 +1,34 @@
+Description: The scripts are a bit fragile
+ REVISION results in lots of garbage if git is not installed (or no .git
+ directory is found) and is also evaluated for each source file instead
+ of once.
+ .
+ os-release calls awk with -c, which isn't necessary (and not supported
+ by some awks).
+ .
+ nextspace-applications (0.85+git7f12a6e-1) unstable; urgency=medium
+ .
+   * Initial release
+Author: Patrick Georgi <patrick@georgi.software>
+
+---
+Last-Update: 2020-05-19
+
+--- nextspace-applications-0.85+git7f12a6e.orig/Workspace/GNUmakefile
++++ nextspace-applications-0.85+git7f12a6e/Workspace/GNUmakefile
+@@ -8,7 +8,7 @@ include $(GNUSTEP_MAKEFILES)/common.make
+ 
+ PACKAGE_NAME = Workspace
+ VERSION = 0.8
+-REVISION = "`git log -n 1 --date=short --format=format:"%h, %ad"`"
++REVISION := "$(shell git log -n 1 --date=short --format=format:"%h, %ad" 2>/dev/null || echo unknown)"
+ APP_NAME = Workspace
+ OS_NAME=`./os-release`
+ $(APP_NAME)_APPLICATION_ICON = app-$(OS_NAME)-48.tiff
+--- nextspace-applications-0.85+git7f12a6e.orig/Workspace/os-release
++++ nextspace-applications-0.85+git7f12a6e/Workspace/os-release
+@@ -1,3 +1,3 @@
+ #!/bin/sh
+ #echo `grep ^NAME= /etc/os-release | awk -F\" -c '{print $2}' | awk -c '{print $1}'`
+-echo `grep ^ID= /etc/os-release | awk -F\= -c '{print $2}' | tr -d \"`
++echo `grep ^ID= /etc/os-release | awk -F\= '{print $2}' | tr -d \"`
```

Comment:
If it's not in master, please create a pull request.

Reply:
it's in master so this can be dropped with the next uprev.